src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -4895,9 +4896,21 @@ void MusicXmlParserDirection::bracket(const String& type, const int number,
             TextLine* textLine = toTextLine(sline);
             // if (placement.empty()) placement = "above";  // TODO ? set default
 
-            textLine->setBeginHookType(lineEnd != "none" ? HookType::HOOK_90 : HookType::NONE);
+            if (configuration()->importLayout() && !endLength.empty()) {
```

Comment:
I'm not so sure about treating this as "layout" information - I'm not sure what "layout" even refers to as it is barely used in import and very inconsistently used in export. Currently it controls import of values in the `<default>` tag and two slur/tie properties. I think it needs to either be expanded to _a lot_ of other element's properties or kept to just the `<defaults>` tag. 
I think the latter would be better, as we now have `Format > Reset entire score to default layout`.

Reply:
Agreed, I dropped the corresponding commit.